README.rst

```diff
@@ -90,3 +90,9 @@ Testing checklist
 * If you are writing a Rule, add the test cases for your rule to ``specCases`` in ``test/spec/rule_spec.js``.
 
 * If needed, any new integration tests can go in ``test/spec/integration_spec.js``.
+
+
+Publishing a New Version
+========================
+
+When you have merged changes, update the version number in ``package.json``, and publish the new version to npm using ``npm publish``.
```

Comment:
Here's some thoughts about other things you might want to add:
 - do you need thumbs on the PR to update the version number? Do you include the version number bump in your original PR?
 - we use semantic versioning: http://semver.org/
 - if you need NPM permissions, send a request to front-end-dev@edx.org
 - it looks like we don't create a new release except for Open edX tagging purposes, so it might be worth calling that out